index.js

```diff
@@ -67,6 +67,40 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener("click", e => {
+    toggleColor(e.target)
+})
+
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
+
+likeButton.addEventListener("click", () => {
+    traveler.likes += 1
```

Comment:
This works, but remember we don't want the likes to persist. They should only be cosmetically updated on the page, but if we refresh they should go back to the original number. Bearing that in mind, what else do you think we'd need here to accomplish this? (hint: you'll have to define another element on the page)